discord/utils.py

```diff
@@ -479,6 +479,36 @@ def resolve_template(code):
 
 _MARKDOWN_ESCAPE_REGEX = re.compile(r'(?P<markdown>%s|%s)' % (_MARKDOWN_ESCAPE_SUBREGEX, _MARKDOWN_ESCAPE_COMMON), re.MULTILINE)
 
+def delete_markdown(text, *, ignore_links=True):
+    r"""A helper function that deletes Discord's markdown.
+    Parameters
+    -----------
+    text: :class:`str`
+        The text to delete markdown from.
+    ignore_links: :class:`bool`
+        Whether to leave links alone when removing markdown. For example,
+        if a URL in the text contains characters such as ``_`` then it will
+        be left alone.
+        Defaults to ``True``.
+    Returns
+    --------
+    :class:`str`
+        The text with the markdown special characters deleted.
+    """
+
+    url_regex = r'(?P<url><[^: >]+:\/[^ >]+>|(?:https?|steam):\/\/[^\s<]+[^<.,:;\"\'\]\s])'
```

Comment:
If this is being reused it's probably better to refactor it as a global constant.